pkg/asset/machines/master.go

```diff
@@ -122,31 +122,59 @@ func awsDefaultMasterMachineType(installconfig *installconfig.InstallConfig) str
 
 // Generate generates the Master asset.
 func (m *Master) Generate(dependencies asset.Parents) error {
+	ctx := context.TODO()
 	clusterID := &installconfig.ClusterID{}
-	installconfig := &installconfig.InstallConfig{}
+	installConfig := &installconfig.InstallConfig{}
 	rhcosImage := new(rhcos.Image)
 	mign := &machine.Master{}
-	dependencies.Get(clusterID, installconfig, rhcosImage, mign)
+	dependencies.Get(clusterID, installConfig, rhcosImage, mign)
 
-	ic := installconfig.Config
+	ic := installConfig.Config
 
 	pool := ic.ControlPlane
 	var err error
 	machines := []machineapi.Machine{}
 	switch ic.Platform.Name() {
 	case awstypes.Name:
+		var subnets map[string]string
```

Comment:
we should `subnets := map[string]string{}` or else the https://github.com/openshift/installer/pull/2477/commits/b8c8a596b4925d562f1731086b66c194ec3168ef#diff-66b4febb36af0238bb6c09e72c1c44ecR145 will create error `panic: assignment to entry in nil map`

Reply:
same for worker.go

Reply:
> we should `subnets := map[string]string{}` ...

Fixed in a8b7a9daf -> b27d6333e.